lib/executor/ramping_arrival_rate.go

```diff
@@ -471,3 +463,57 @@ func (varr RampingArrivalRate) Run(parentCtx context.Context, out chan<- stats.S
 	}
 	return nil
 }
+
+// activeVUPool controls the activeVUs
+// executing the received requests for iterations.
+type activeVUPool struct {
+	iterations chan struct{}
+	running    uint64
+	wg         sync.WaitGroup
+}
+
+// newActiveVUPool returns an activeVUPool.
+func newActiveVUPool() *activeVUPool {
+	return &activeVUPool{
+		iterations: make(chan struct{}),
+	}
+}
+
+// Run invokes a request to execute a new iteration.
+// When there aren'p *available VUs to process the request
```

Comment:
```suggestion
// TryRunIteration invokes a request to execute a new iteration.
// When there are no available VUs to process the request
```

We might also unexport all the `activeVUPool` methods, but it's not a big deal.